index.html

```diff
@@ -120,5 +120,40 @@ <h5>Next level iceland blue bottle fingerstache:</h5>
         </ul>
       </section>
     </section>
+    <section class="contact">
+      <h2>contact</h2>
+      <form>
+        <label for="name">name</label>
+        <input type="text" id="name">
+        <label for="email">e-mail</label>
+        <input type="email" id="email">
+        <label for="topic">topic</label>
+        <input type="text" id="topic">
+      </form>
+      <img src="assets/office.jpg" alt="office building image">
+      <p>
+        Our office is located<br/>
+        in the heart of Warsaw
+      </p>
+      <p>Feel free to say hello!</p>
+      <p>
+        Warsaw St. 23/21<br/>
+        03-323 Warsaw,<br/>
+        POLAND
+      </p>
+    </section>
+    <footer>
+      <!-- assets/socialmedia.svg as a background image for footer -->
+      <ul>
+        <li><a href="https://pinterest.com"><img src="assets/pinterest.svg" alt="pinterest logo"></a></li>
```

Comment:
For icons that are not giving much info I am usually hiding the icon itself. Adding a link without any text is harmful as person that has to use screen reader will not be able to find out what the link is about. I suggest:
`<a href="/">Pinterest<img src="assets/pinterest.svg" aria-hidden="true" /></a>`. `aria-hidden` will tell the screen reader to ignore the element altogether.